openshift-kube-apiserver/handlers/deprecatedapirequest/handler.go

```diff
@@ -0,0 +1,30 @@
+package deprecatedapirequest
```

Comment:
funny that the handler.go file does not contain the handler.

Reply:
I moved the contents to a more appropriate location.